lib/locales/es.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
         'Click to enter Component B title': 'Introducir el título del Componente B', // plots/ternary/ternary.js:406
         'Click to enter Component C title': 'Introducir el título del Componente C', // plots/ternary/ternary.js:417
         'Click to enter Plot title': 'Introducir el título de la Gráfica', // plot_api/plot_api.js:579
+        'Click to enter Plot subtitle': 'Introducir el subtítulo de la Gráfica', // plot_api/plot_api.js:579
```

Comment:
Thanks for adding these translations.
The line numbers and filenames in comments may not be accurate.
Let's open a PR and remove them from few files in `lib/locales` folder that have those after merging this PR.
Thank you!

Reply:
Issue opened: https://github.com/plotly/plotly.js/issues/7040